src/Myra/Graphics2D/UI/Window.cs

```diff
@@ -121,6 +122,10 @@ public bool Result
 			get; set;
 		}
 
+		[Category("Appearance")]
```

Comment:
Can you, please, rename it to "Behavior"?

Reply:
Done